website/docs/cli/configuration/markdown-styling.mdx

```diff
@@ -0,0 +1,204 @@
+---
+title: Markdown Styling
+sidebar_label: Markdown Styling
+sidebar_position: 7
+description: Configure custom markdown styling for Atmos CLI output
+---
+
+# Markdown Styling Configuration
```

Comment:
Does this also relate to `atmos docs` which uses Glamour?

Reply:
website uses docusaurus and CLI uses glamour

Reply:
@Cerebrovinny I mean the `atmos docs` subcommand that was previously implemented and already uses Glamour.